saos-webapp/src/test/java/pl/edu/icm/saos/webapp/judgment/detail/JudgmentDetailsSortServiceTest.java

```diff
@@ -0,0 +1,75 @@
+package pl.edu.icm.saos.webapp.judgment.detail;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import pl.edu.icm.saos.persistence.model.CommonCourtJudgment;
+import pl.edu.icm.saos.persistence.model.Judge;
+
+/**
+ * 
+ * @author Łukasz Pawełczak
+ *
+ */
+public class JudgmentDetailsSortServiceTest {
+
+	
+	private JudgmentDetailsSortService judgmentDetailsSortService = new JudgmentDetailsSortService();
+	
+	//------------------------ TESTS --------------------------
+	
+	@Test
+	public void sortJudges_simple() {
+		
+		//given
+		Judge judgeOne = new Judge("Kamil Graczyk", Judge.JudgeRole.REPORTING_JUDGE);
+		Judge judgeTwo = new Judge("Marzena Kisiela", Judge.JudgeRole.PRESIDING_JUDGE);
+		
+		CommonCourtJudgment unsortedJudgment = new CommonCourtJudgment();
+		
+		unsortedJudgment.addJudge(judgeOne);
+		unsortedJudgment.addJudge(judgeTwo);
+		
+		//execute
+		
+		CommonCourtJudgment sortedJudgment = (CommonCourtJudgment)judgmentDetailsSortService.sortJudges(unsortedJudgment);
```

Comment:
It is irrelevant in this case what kind of judgment is returned. You could remove casting and assign it to variable with Judgment type